public/app/core/services/context_srv.ts

```diff
@@ -77,7 +77,7 @@ export class ContextSrv {
   }
 
   // Checks whether user has required permission
-  hasPermission(action: AccessControlAction): boolean {
+  hasPermission(action: AccessControlAction | string): boolean {
```

Comment:
Ok I see the issue now. Basically we're losing all the type support here since we don't get an error when actions is not an `AccessControlAction`. Not sure what's the best approach here 🤔

Reply:
Yes. I don't think it's a big problem, anyway we're using constants, not just a strings like `reports.admin:write`.